### PR TITLE
feat: provide more logging and signature

### PR DESCRIPTION
**Pull request by EstebanBorai**

### CODE REVIEW

**Comment by morenol** [comment]
@EstebanBorai crate version for this is 0.0.0 https://github.com/infinyon/sql-connector/blob/80eae06810dc7c9b94bb6ac43c1168d41635356f/crates/sql-sink/Cargo.toml#L3

Can we add a VERSION file at the root(similar to how we do it for fluvio repo) of the repo and use it instead

**Comment by ozgrakkurt** [comment]
can we merge this?

**Comment by morenol** [comment]
in the current shape, CARGO_PKG_VERSION is 0.0.0 which I think that is not useful

File: crates/sql-sink/src/sink.rs [hunk 1]
```diff
@@ -25,11 +25,12 @@ impl SqlSink {
 impl Sink<Operation> for SqlSink {
     async fn connect(self, _offset: Option<Offset>) -> Result<LocalBoxSink<Operation>> {
         let db = Db::connect(self.url.as_str()).await?;
-        info!("connected to database {}", db.kind());
+        info!("Connected to Database {}", db.kind());
```
Comment (sehz) [inline crates/sql-sink/src/sink.rs:28]: use `info!(db.kind,"Connected to Database")`

File: crates/sql-sink/src/main.rs [hunk 2]
```diff
@@ -6,19 +6,25 @@ mod sink;
...
 use sink::SqlSink;
 
+const SIGNATURES: &str = concat!("InfinyOn SQL Sink Connector ", env!("CARGO_PKG_VERSION"));
+
 #[connector(sink)]
 async fn start(config: SqlConfig, mut stream: impl ConsumerStream) -> Result<()> {
     let sink = SqlSink::new(&config)?;
     let mut sink = sink.connect(None).await?;
+
+    tracing::info!("Starting {SIGNATURES}");
```
Comment (sehz) [inline crates/sql-sink/src/main.rs:21]: let's move common connector startup logging to fluvio

**Pull request closed without merging**